.github/workflows/semver-checks.yml

```diff
@@ -3,34 +3,85 @@ on:   # yamllint disable-line rule:truthy
 
 name: Check semver breaks
 
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
Needs to implement the suggestion from this article instead: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
Have two jobs, first one does the check and uploads an artifact, the second one gets the artifact and writes the comment if the artifact says "break".